cancermuts/table.py

```diff
@@ -165,7 +165,8 @@ def __init__(self, labels=None, ptm_colors=None, y_ptm=1.02):
         headers['position']    = SequencePosition.header
         headers['wt']          = 'ref_aa'
         headers['mutated']     = 'alt_aa'
-        headers['mut_sources'] = 'sources'
+        headers['mut_sources'] = 'mut_sources'
```

Comment:
can we keep this `sources` as is?

just because it potentially breaks compatibility with tools that use the metatable